### DISCUSSION TITLE
Argon skin

### DISCUSSION THREAD

**Discussion by Dominilk01** [ppy/osu#28797] [Q&A]
is there a way to make that argon don't have the green part ?
![image](https://github.com/ppy/osu/assets/108392199/7f8fb0d1-adc3-41c5-9fc6-18846c68a69a)

**Comment by Dominilk01** [comment]
What i mean is there a option to remove the bettewn the sliderfollowcircle and hitcircle

**Comment by Stoppedpuma** [comment]
As of now, no. This is planned though! See https://github.com/ppy/osu/discussions/28070#discussioncomment-9297577.
